### PR TITLE
fix slashing e2e tests

### PR DESCRIPTION
**Pull request by aslawson**
### Description

fix double signing slashing tests -- changed config slashing penalty from 5k to 9k

### Tested

NA

### Other changes

NA

### Related issues

follow up to https://github.com/celo-org/celo-monorepo/pull/2972 change

### Backwards compatibility

NA

### CODE REVIEW

File: packages/celotool/src/e2e-tests/slashing_tests.ts [hunk 1]
```diff
@@ -314,7 +314,7 @@ describe('slashing tests', function(this: any) {
 
       // Penalty is defined to be 5000 cGLD in migrations, locked gold is 10000 cGLD for a validator
       const balance = await lockedGold.getAccountTotalLockedGold(signer)
-      assert.equal(balance.toString(10), '5000000000000000000000')
+      assert.equal(balance.toString(10), '1000000000000000000000')
```
Comment (lucasege) [inline packages/celotool/src/e2e-tests/slashing_tests.ts]: Nit: change comment to 9k penalty, and add ~ "so after slashing locked gold is 1000cGld"

File: packages/celotool/src/e2e-tests/slashing_tests.ts [hunk 2]
```diff
@@ -347,7 +347,7 @@ describe('slashing tests', function(this: any) {
       // Penalty is defined to be 5000 cGLD in migrations, locked gold is 10000 cGLD for a validator
       const lockedGold = await kit.contracts.getLockedGold()
       const balance = await lockedGold.getAccountTotalLockedGold(signer)
-      assert.equal(balance.toString(10), '5000000000000000000000')
+      assert.equal(balance.toString(10), '1000000000000000000000')
```
Comment (lucasege) [inline packages/celotool/src/e2e-tests/slashing_tests.ts]: Same as above

**Review by lucasege — APPROVED** [review]
lgtm, 1 nit on a comment change

File: packages/celotool/src/e2e-tests/slashing_tests.ts (hunk 1 above)
Comment (aslawson) [reply packages/celotool/src/e2e-tests/slashing_tests.ts]: good catch, thanks!

**Pull request merged**